rwengine/src/engine/GameWorld.cpp

```diff
@@ -1022,4 +1022,13 @@ void GameWorld::clearObjectsWithinArea(const glm::vec3 center,
 
     // @todo Remove all temp objects, extinguish all fires, remove all
     // explosions, remove all projectiles
+}
+
+PlayerController* GameWorld::getPlayer() {
```

Comment:
When can this method fail? In other words, why wouldn't there always be a player when it's called, could it not return a reference instead and throw if one isn't found?